src/dependency-manager/src/spec/common/volume-v1.ts

```diff
@@ -11,10 +11,6 @@ export class VolumeSpecV1 extends ValidatableConfig {
     groups: ['debug'],
     message: 'Debug volumes require a host path to mount the volume to',
   })
-  @IsEmpty({
```

Comment:
I don't think we can remove this check. The host_path won't exist for remote deploys. There is an open issue for next milestone to address.
https://gitlab.com/architect-io/architect-cli/-/issues/116#note_473026607

Reply:
Sure, I've added redundant debug volumes to our configs in the meantime